microsoft-edge/webview2/release-notes/about.md

```diff
@@ -0,0 +1,113 @@
+---
+title: About Release Notes for the WebView2 SDK
+description: About Release Notes for Microsoft Edge WebView2, covering new features, APIs, and fixes for Win32, WPF, and WinForms.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: webview
+ms.date: 03/18/2024
+---
+# About Release Notes for the WebView2 SDK
+
+The WebView2 team updates the WebView2 SDK on a four-week cadence.  This article contains the latest information on product announcements, additions, modifications, and breaking changes to the APIs.
+
+You can view the list of [Microsoft.Web.WebView2](https://www.nuget.org/packages/Microsoft.Web.WebView2) SDK packages at the NuGet site.
+
+Generally, release notes apply across the supported platforms, which are listed in [WebView2 API Reference](../webview2-api-reference.md).  For an outline of APIs that are in Release SDK packages, see [Overview of WebView2 features and APIs](../concepts/overview-features-apis.md).
+
+Release Notes entries correspond to historical releases of WebView2, and are not updated over time.  References to "new features" and "experimental APIs" might become outdated as new versions of WebView2 are released.
+
+
+<!-- ====================================================================== -->
+## Phases of adding APIs
+
+![Phases of adding APIs](./about-images/phases-of-adding-apis.png)
+
+The WebView2 team updates the WebView2 SDK on a four-week cadence.  This article contains the latest information on product announcements, additions, modifications, and breaking changes to the APIs.
+
+You can view the list of [Microsoft.Web.WebView2](https://www.nuget.org/packages/Microsoft.Web.WebView2) SDK packages at the NuGet site.
+
+Generally, release notes apply across the supported platforms, which are listed in [WebView2 API Reference](../webview2-api-reference.md).  For an outline of APIs that are in Release SDK packages, see [Overview of WebView2 features and APIs](../concepts/overview-features-apis.md).
+
```

Comment:
The following two sections in the about page are duplicated information and diagram, and should be combined into just one section. Let me know if you would need me to make a commit for this change


![image](https://github.com/MicrosoftDocs/edge-developer/assets/5773562/71e9e51f-ca7b-420f-af07-b28cf25716a1)
![image](https://github.com/MicrosoftDocs/edge-developer/assets/5773562/57fc77f5-dd21-4fc8-9379-3f5ca931995d)

Reply:
resolved the dup of 3 paragraphs at top.  Combined/cleaned up the two sections into one.